homeassistant/components/sensor/zwave.py

```diff
@@ -33,26 +34,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     # if 1 in groups and (NETWORK.controller.node_id not in
     #                     groups[1].associations):
     #     node.groups[1].add_association(NETWORK.controller.node_id)
+    name = '{}.{}'.format(DOMAIN, zwave.object_id(value))
+    node_config = customize.get_overrides(hass, zwave.DOMAIN, name)
```

Comment:
As long as we can guarantee that the reports from Z-Wave sensors is the current state and not a cached state, I am down for setting `force_update` to True for sensors.